lib/grape-swagger/endpoint.rb

```diff
@@ -43,12 +43,12 @@ def swagger_object(target_class, request, options)
     # building info object
     def info_object(infos)
       result = {
-        title:             infos[:title] || 'API title',
-        description:       infos[:description],
-        termsOfServiceUrl: infos[:terms_of_service_url],
-        contact:           contact_object(infos),
-        license:           license_object(infos),
-        version:           infos[:version]
+        title: infos[:title] || 'API title',
+        description: infos[:description],
+        termsOfService: infos[:terms_of_service_url],
```

Comment:
wow, this is in since year, and nobody saw that 😉

Reply:
probably nobody cares about TOS ;)